cna-handbook.md

```diff
@@ -96,64 +99,66 @@ To see help output on creating a new CNA admin user, you can run the following c
-This should be done as soon as reasonably possible after the vulnerability is made public (ideally, O(days)).
-
-See https://github.com/CVEProject/cvelist/pull/3176/files as an example.
-
-https://vulnogram.github.io/#editor is a useful tool for editing the CVE details,
-but should only be used once the vulnerability has been made public.
-
-Fill in or update the relevant fields, including:
-* State: `PUBLIC`
-* Assigner: `security@kubernetes.io`
-* Title: `Kubernetes <component> <vulnerability>`
-* Affects: List historical minor versions affected and "prior to 1.x.y" patch versions affected
-* Credit: The vulnerability reporter's name
-* Description: `The Kubernetes <component> command/component in versions <affected versions> <vulnerability>`
-* Impact: CVSS impact
-* Problem type: Select a problem type from https://cwe.mitre.org/data/definitions/699.html if applicable
-* References: Link to the Kubernetes github issue(s) (with type CONFIRM) and mailing list announcement (with type MLIST)
-* Source: Link to the Kubernetes github issue(s) and indicate if it was discovered internally or by a user
-* Workaround: indicate workaround steps, if applicable
-
-Commit, push to your branch, create a pull request to
-https://github.com/kubernetes-security/cvelist-public master, and link the pull request in the
-[tracking sheet]. Review any errors from the PR validation, and request a review from a CNA-approved
-SRC member.
-
-Once the PR to our fork has merged, a CNA approved member should open a PR to sync the upstream
-cvelist with our fork:
-https://github.com/CVEProject/cvelist/compare/master...kubernetes-security:master
-
-
-Once the upstream pull request is merged, indicate in the tracking sheet that the CVE has been
-populated.
+1. Once a vulnerability is made public, populate the CVE details in https://github.com/CVEProject/cvelist. This should be done as soon as reasonably possible after the vulnerability is made public (ideally, O(days)).
+1. Generate the CVElist json file
+
+    * See https://github.com/CVEProject/cvelist/blob/master/2023/2xxx/CVE-2023-2727.json as an example.
+
+    * https://vulnogram.github.io/#editor is a useful tool for generating the CVE details, but should only be used once the vulnerability has been made public.
+
+    Fill in or update the relevant fields, including:
+    * State: `PUBLIC`
+    * Assigner: `security@kubernetes.io`
+    * Affects: List historical minor versions affected and "prior to 1.x.y" patch versions affected
+    * Credits: The vulnerability reporter's name
+    * Description: `The Kubernetes <component> command/component in versions <affected versions> <vulnerability>`
+    * Impact: CVSS impact
+    * Problem type: Select a problem type from https://cwe.mitre.org/data/definitions/699.html if applicable
+    * References: Link to the Kubernetes github issue(s) (with type CONFIRM) and mailing list announcement (with type MLIST)
```

Comment:
```suggestion
    * References: Link to the Kubernetes github issue(s) (with type `issue-tracking`) and mailing list announcement (with type `mailing-list`)
```
running through this now and the tags in the tool may have changed.

Reply:
updated